infrastructure/modules/worker/main.tf

```diff
@@ -1,13 +1,15 @@
 locals {
   # Override the default service name if requested
   deployment_service_name = var.deployment_service_name == "" ? var.name : var.deployment_service_name
+
+  service_name = var.service_name == "" ? var.name : var.service_name
```

Comment:
I'm struggling to understand this, or the other related changes. Would it make sense to add a comment explaining?

Reply:
I've added a comment, any better?

Reply:
Sort of.. doesn't that make it clash with other pipelines though?

Reply:
No:

* In the ECS console, service names only have to be unique within a cluster. You can repeat names across clusters.
* In the logging cluster, you can filter by (cluster, service name) to find just those services – but now you can actually find the service name.